lib/tasks/demo.rake

```diff
@@ -900,89 +897,97 @@ namespace :demo do
   end
 
   def demo_populate_linelists
-    # populate :latest_assessment_at
-    ActiveRecord::Base.connection.execute <<-SQL.squish
-      UPDATE patients
-      INNER JOIN (
-        SELECT patient_id, MAX(created_at) AS latest_assessment_at
-        FROM assessments
-        GROUP BY patient_id
-      ) t ON patients.id = t.patient_id
-      SET patients.latest_assessment_at = t.latest_assessment_at
-    SQL
-
-    # populate :latest_assessment_symptomatic
-    ActiveRecord::Base.connection.execute <<-SQL.squish
-      UPDATE patients
-      JOIN (
-        SELECT assessments.patient_id
-        FROM assessments
-        JOIN (
+    ActiveRecord::Base.transaction do
```

Comment:
@timwongj please review just these demo.rake changes

Reply:
@holmesie Was that change from another PR that was already merged into 1.32.0? I don't see any changes to demo.rake in the diff

Reply:
Most likely, it seems to have gone away with the rebase.